app/src/org/commcare/android/view/TabbedDetailView.java

```diff
@@ -159,7 +166,7 @@ public void refresh(Detail detail, TreeReference reference, int index, boolean h
                 hasDetailCalloutListener, new DefaultEDVModifier()
         );
         mViewPager.setAdapter(mEntityDetailPagerAdapter);
-        markSelectedTab(0);
+        if(!useNewTabStyle) markSelectedTab(0);

```

Comment:
Why is this line conditional on `useNewTabStyle`, but not the other places that the first tab is selected by default?